lib/nexmo_developer/app/views/layouts/partials/_head.html.erb

```diff
@@ -3,9 +3,7 @@
 <head>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  
-  <script src="https://assets.adobedtm.com/54d668d6379a/4bfed685397d/launch-029c4ab10638-development.min.js" async></script>
- 
+  <script src='<%= ENV['ADOBE_LAUNCH_URL'] %>' async></script>
```

Comment:
What happens if that ENV variable doesn't exist?

Reply:
This is the URL required for Adobe Analytics to set data layer variables. If it does not exist, the analytics will not work. We need to include ADOBE_LAUNCH_URL='https://assets.adobedtm.com/54d668d6379a/4bfed685397d/launch-1b04eb013f27.min.js' in Heroku environment.

Reply:
but that means we'll have a script tag with no src and we'll get penalised by the SEO

Reply:
I am confused here, I understand that there is a production url configured in Heroku environment as a ENV variable. Could you please provide the ENV variable name so that I'll use it here.